mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.tmpl

```diff
@@ -4110,85 +4111,84 @@ resource "google_container_cluster" "cluster" {
     services_secondary_range_name = google_compute_subnetwork.container_subnetwork.secondary_ip_range[1].range_name
   }
   release_channel {
-	channel = "RAPID"
+    channel = "RAPID"
   }
   deletion_protection = false
 }
 
 resource "google_container_node_pool" "with_manual_pod_cidr" {
-  name               = "%s-manual"
-  location           = "us-central1"
-  cluster            = google_container_cluster.cluster.name
+  name       = "%s-manual"
+  location   = "us-central1-c"
```

Comment:
@slevenick not sure if it would be best / cleaner to use `google_container_cluster.cluster.location` in all of these if it works? obviously there's already an implicit dependency already with the cluster name, but it would keep them consistent without having to declare a variable. 🤷 
Happy to update it to that if you think it would be better... I think it will work either way.

Reply:
Yeah, that sounds reasonable. Then updating the location will be super simple in the future

Reply:
Ok, updated. Trying a local test with the two node pools that were causing me issues commented out.